xarray/tests/test_dask.py

```diff
@@ -499,11 +499,11 @@ def test_dataarray_repr(self):
         a = DataArray(data, dims=["x"], coords={"y": ("x", nonindex_coord)})
         expected = dedent(
             """\
-        <xarray.DataArray 'data' (x: 1)>
-        dask.array<shape=(1,), dtype=int64, chunksize=(1,)>
-        Coordinates:
-            y        (x) int64 dask.array<shape=(1,), chunksize=(1,)>
-        Dimensions without coordinates: x"""
+            <xarray.DataArray 'data' (x: 1)>
+            dask.array<data, shape=(1,), dtype=int64, chunksize=(1,)>
+            Coordinates:
+                y        (x) int64 dask.array<shape=(1,), chunksize=(1,)>
+            Dimensions without coordinates: x"""
```

Comment:
Why would I want to know the dask name for the data, but not for the coords? for me it's either both or neither.
I'm in favour of neither because actual real-life dask names can get VERY long!